.github/workflows/ci.yml

```diff
@@ -54,15 +51,13 @@ jobs:
         with:
           python-version: "3.12.1"
 
-      - name: Set up KLC Check
-        # FIXME: Temporarily pinned to v8 to pass tests
-        run: |
-          git clone --depth 1 --branch v8 https://gitlab.com/kicad/libraries/kicad-library-utils /opt/kicad-library-utils
-          /opt/kicad-library-utils/klc-check/check_symbol.py --help
-          /opt/kicad-library-utils/klc-check/check_footprint.py --help
+      - name: Set up KiCad Library Utils
+        uses: loozhengyuan/actions/setup-kicad-library-utils
+        with:
+          version: v8
```

Comment:
- The `v8` input should be quoted
- Can be restore the comment `FIXME: Temporarily pinned to v8 to pass tests` to help provide context on why this is needed?
- Can you do the same for `test-symbols` job as well
- I think the `Set up Python` step is probably redundant, we can remove it from both jobs.